pkgs/by-name/cl/clock-tui/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  installShellFiles,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "clock-tui";
+  version = "0.5.0";
+  src = fetchFromGitHub {
+    owner = "race604";
+    repo = "clock-tui";
+    tag = "v${version}";
+    hash = "sha256-sXfGm36WgL6h06cT9TowkaEaBB1roe4qP3OW2JJ3EwQ=";
+  };
+
+  cargoHash = "sha256-oiQ8RiCPP2PCQv+oUb299ijWZHL1xR13/UPvVeMojYQ=";
```

```suggestion
  useFechCargoVendor = true;
  cargoHash = "sha256-oiQ8RiCPP2PCQv+oUb299ijWZHL1xR13/UPvVeMojYQ=";
```
(this will change the hash)